src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_reader.py

```diff
@@ -1,32 +1,59 @@
 from __future__ import annotations
 
+from dataclasses import dataclass
+from io import StringIO
 import re
 from typing import Any
 
 import pandas as pd
 
 from allotropy.allotrope.pandas_util import read_excel
+from allotropy.named_file_contents import NamedFileContents
 from allotropy.parsers.beckman_vi_cell_xr.constants import (
     DATE_HEADER,
     DEFAULT_VERSION,
     HEADINGS_TO_PARSER_HEADINGS,
     MODEL_RE,
     XrVersion,
 )
+from allotropy.parsers.lines_reader import read_to_lines
 from allotropy.parsers.utils.pandas import SeriesData
 from allotropy.parsers.utils.values import assert_value_from_df
-from allotropy.types import IOType
+
+
+@dataclass
```

Comment:
Note to reviewer: I added this little refactor because I noticed that the `structure` class isn't actually using the reader except to access these two pieces of data, so just return them.